packages/create-amplify/src/create_amplify.ts

```diff
@@ -16,17 +16,23 @@ import { TsConfigInitializer } from './tsconfig_initializer.js';
 import { getProjectRoot } from './get_project_root.js';
 import { PackageJsonReader } from './package_json_reader.js';
 import { GitIgnoreInitializer } from './gitignore_initializer.js';
+import { logger } from './logger.js';
 
 const projectRoot = await getProjectRoot();
 
 const amplifyProjectCreator = new AmplifyProjectCreator(
   new NpmPackageManagerController(projectRoot),
   new ProjectRootValidator(projectRoot),
   new InitialProjectFileGenerator(projectRoot),
-  new NpmProjectInitializer(projectRoot),
-  new TsConfigInitializer(projectRoot, new PackageJsonReader(projectRoot)),
-  new GitIgnoreInitializer(projectRoot),
-  projectRoot
+  new NpmProjectInitializer(projectRoot, logger),
+  new TsConfigInitializer(
+    projectRoot,
+    new PackageJsonReader(projectRoot),
+    logger
+  ),
```

Comment:
If we are using static instance of loggers all over the place, do we still need to pass it in the ctors? We should be able to mock that instance right?

Reply:
Done in [fb89cac](https://github.com/aws-amplify/amplify-backend/pull/595/commits/fb89cacbc4748373aa55231303d37aa8d05dccbc)